packages/react/src/components/sideNavigation/subLevel/SubLevel.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useEffect } from 'react';
+
+import styles from './SubLevel.module.scss';
+import SideNavigationContext from '../SideNavigationContext';
+import classNames from '../../../utils/classNames';
+
+export type SubLevelProps = React.ComponentPropsWithoutRef<'a'> & {
+  /**
+   * If `true`, the item will be marked as active
+   */
+  active?: boolean;
+  /**
+   * Additional class names to apply to the side navigation sub level
+   */
+  className?: string;
+  /**
+   * The id of the side navigation sub level.
+   */
+  id: string;
+  /**
+   * href attribute of the side navigation sub level
+   */
+  href: string;
+  /**
+   * Label of the side navigation sub level.
+   */
+  label: string;
+  /**
+   * Index of the containing main level
+   */
+  mainLevelIndex?: number;
```

Comment:
Should we add a comment that this is generated automatically by the main level component, and user of the SubLevel should not provide this by themself?

Reply:
Yes 👍 
- [ ] add clarifying comment of mainLevelIndex